config/src/config/state_sync_config.rs

```diff
@@ -123,10 +123,10 @@ impl Default for StorageServiceConfig {
             max_epoch_chunk_size: 200,
             max_lru_cache_size: 500, // At ~0.6MiB per chunk, this should take no more than 0.5GiB
             max_network_channel_size: 4000,
-            max_network_chunk_bytes: MAX_MESSAGE_SIZE as u64,
+            max_network_chunk_bytes: 20971520,
             max_state_chunk_size: 4000,
             max_subscription_period_ms: 5000,
-            max_transaction_chunk_size: 2000,
+            max_transaction_chunk_size: 5000,
```

Comment:
Technically, we want to update `max_transaction_output_chunk_size` to 5000. But, let's do both (in case we ever need to switch over to execution mode).